CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.30.0)
   cmake_policy(SET CMP0167 OLD)
 endif()
 
+# use gnu standard install directories
+include(GNUInstallDirs)
+set(ARBORX_INSTALL_PACKAGEDIR "${CMAKE_INSTALL_LIBDIR}/cmake/ArborX" CACHE PATH "Install location of CMake target files")
```

Comment:
I always thought `${CMAKE_INSTALL_DATADIR}/cmake/${PROJECT_NAME}` was the default, but https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure is not clear about that.

And in their tutorial they are using `lib` as well, https://cmake.org/cmake/help/book/mastering-cmake/cmake/Help/guide/tutorial/index.html#adding-export-configuration-step-11